src/components/icons.ts

```diff
@@ -62,7 +66,7 @@ export const centeredSpinner = ({ size = 48, ...props } = {}) =>
     )
   );
 
-export const wdlIcon = ({ style = {}, ...props } = {}) =>
+export const wdlIcon = ({ style = {}, ...props }: JSX.IntrinsicElements['div'] = {}) =>
```

Comment:
`JSX.InstrinsicElements['div']` is a type for the props accepted by a `div` element.